python/py-google-auth-httplib2/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-google-auth-httplib2
+version             0.0.3
+
+python.versions     27 34 35 36 37
+
+categories-append   www devel
+maintainers         {wholezero.org:macports @mrdomino} openmaintainer
+license             Apache-2
+description         provides an httplib2 transport for google-auth
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://pypi.python.org/pypi/${python.rootname}
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  2a244e22de54eac2042adb13b87dc39001b77084 \
+                    sha256  098fade613c25b4527b2c08fa42d11f3c2037dda8995d86de0745228e965d445 \
+                    size    9957
+
+if {${name} ne ${subport}} {
+    depends_build       port:py${python.version}-setuptools
+    depends_lib-append  port:py${python.version}-google-auth \
+                        port:py${python.version}-httplib2 \
+                        port:py${python.version}-six
```

Comment:
from the ```setup.py``` it looks like ```six``` is not an actual dependency. It appears to be only used in the tests, but since the test phase is not included in the Portfile, you should not include them here unless there is a reason I am missing?